handlers/zuora-callout-apis/src/main/scala/com/gu/util/zuora/ZuoraCreditTransfer.scala

```diff
@@ -0,0 +1,71 @@
+package com.gu.util.zuora
+
+import com.gu.util.resthttp.RestRequestMaker.Requests
+import com.gu.util.resthttp.Types.{ClientFailableOp, ClientSuccess}
+import com.gu.util.zuora.ZuoraCreditTransfer.AdjustmentType
+import com.typesafe.scalalogging.LazyLogging
+import play.api.libs.json._
+
+object ZuoraCreditTransfer extends LazyLogging {
+
+  sealed trait AdjustmentType { def value: String }
+  object AdjustmentType {
+    case object Increase extends AdjustmentType { val value = "Increase" }
+    case object Decrease extends AdjustmentType { val value = "Decrease" }
+  }
```

Comment:
simpler to write this way
```suggestion
  sealed abstract class AdjustmentType(val value: String)
  object AdjustmentType {
    case object Increase extends AdjustmentType("Increase")
    case object Decrease extends AdjustmentType("Decrease")
  }
```